app/assets/stylesheets/pages/_roasts.scss

```diff
@@ -0,0 +1,255 @@
+          @include fluid-size('width', $range-handle-size, $range-handle-size-big);
+          @include fluid-size('height', $range-handle-size, $range-handle-size-big);
+        }
+      }
+      .slider__range::before {
+        -webkit-appearance: none;
+        overflow: hidden;
+        color: #13bba4;
+      }
+    }
+    @include breakpoint($large-screens) {
+      order: 1;
+      margin-top: 5%;
+      margin-bottom: 5%;
+    }
+  }
+  .roasts__view {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    order: 1;
+    .view__image-wrapper {
+      display: flex;
+      margin-top: 5vh;
+      margin-bottom: 2vh;
+    }
+
+    .view__text-wrapper {
+      text-align: center;
+      h2 {
+        @include fluid-size('font-size', 24px, 36px);
+        color: $c-brown;
+        font-weight: 400;
+        margin-bottom: 1.4vw;
+      }
+      p {
+        @include fluid-size('font-size', 14px, 18px);
+        margin-bottom: 0.7vw;
+        strong {
+          font-weight: 700;
+        }
+      }
+    }
+
+    @include breakpoint($large-screens) {
+      order: 2;
+      align-self: center;
+      flex-direction: row;
+      align-items: center;
+      justify-content: center;
+      margin-top: 5%;
+      margin-bottom: 5%;
+      width: 80%;
+      .view__image-wrapper {
+        margin: 0;
+      }
+      .view__text-wrapper {
+        display: flex;
+        flex-direction: column;
+        width: 35%;
+        margin-left: 12%;
+      }
+
+    }
+  }
+  .roasts__brands-wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    order: 3;
+    width: 100%;
+    background-color: #F5F5F5;
+    @include fluid-sizeFULL('padding-top', 24px, 40px);
+    .roasts__brands {
+      width: 500px;
+      align-self: center;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      @include breakpoint(min-width 900px) {
+        width: 100%;
+      }
+      h2 {
+        color: $c-text;
+        @include fluid-sizeFULL('margin-bottom', 22px, 44px);
+        @include fluid-sizeFULL('font-size', 24px, 36px);
+      }
+      .brands__wrapper {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: center;
+        margin-bottom: 5%;
+        width: initial;
+        @include breakpoint(min-width 900px) {
+          justify-content: space-around;
+        }
+        @include breakpoint(min-width 1000px) {
+          width: 83%;
+        }
+      }
+      .brand__wrapper {
+        padding: 4px;
+        img {
+          border-radius: 10px;
+        }
+        .brand__text-wrapper {
+          color: white;
+          margin-top: -60%;
+          margin-bottom: 40%;
+          padding-bottom: 6px;
+          text-align: center;
+        }
+      }
+    }
+  }
+  
+
+  h3 {
+    @include fluid-size('font-size', 10px, 14px);
+    font-family: $f-body;
+    margin-bottom: 5px;
+  }
+}
+
+.screen-reader-only {
```

Comment:
@mzlock Weirdly enough, `base/_utilities` wasn't actually linked in our `main.scss`.